src/org/labkey/test/components/html/Table.java

```diff
@@ -92,6 +92,28 @@ public List<String> getTableHeaderTexts()
         return columnHeaders;
     }
 
+    public int getTableHeaderIndex(String headerText)
+    {
+        List<WebElement> headerEls = Locator.xpath("//thead//th").findElements(this);
+        int counter = 1;
+        for(WebElement headerEl : headerEls)
+        {
+            if(headerEl.getText().equalsIgnoreCase(headerText))
+                return counter;
+            counter++;
+        }
+        return counter;
```

Comment:
Don't return an inaccurate value. This should throw an exception if the specified column doesn't exist

Reply:
good point